config/redirects.txt

```diff
@@ -13,3 +13,6 @@
 ^\/getting-started -> /raspberrypi/nodejs/getting-started
 
 ^\/configuration\/resin-base-images(\/|$) -> /runtime/resin-base-images
+
+# Renamed pages
+^\/deployment\/wifi\/?(.*)? -> /deployment/network/$1
```

Comment:
Ah, and you also don't need the outer group:

`^\/deployment\/wifi\/?(#.+)?$` (and then you can use `$1` in the replacement

http://regexr.com/3e4mh